isis/src/base/apps/mappt/mappt.cpp

```diff
@@ -266,91 +282,56 @@ void mappt(Cube *icube, UserInterface &ui, Pvl *log, CubeAttributeInput* inAtt)
         results += pE180;
         results += pW360;
       }
-    }
-    
-    if (log) {
-      log->addGroup(results);
-    }
+    } 
+  }
 
-    // Write an output label file if necessary
-    if(ui.WasEntered("TO")) {
-      // Get user params from ui
-      QString outFile = FileName(ui.GetFileName("TO")).expanded();
-      bool exists = FileName(outFile).fileExists();
-      bool append = ui.GetBoolean("APPEND");
-
-      // Write the pvl group out to the file
-      if(ui.GetString("FORMAT") == "PVL") {
-        Pvl temp;
-        temp.addGroup(results);
-        if(append) {
-          temp.append(outFile);
-        }
-        else {
-          temp.write(outFile);
-        }
-      }
+  return results; 
+}
 
-      // Create a flatfile of the same data
-      // The flatfile is comma delimited and can be imported into Excel
-      else {
```

Comment:
What happened to the flatfile output?

Reply:
I think I removed it intending to put it back in a different place that made more sense, and then forgot to actually do it. I'll make the fix as soon as I can.